### DISCUSSION TITLE
How do I set page rules?

### DISCUSSION THREAD

**Discussion by ethertech-jackd** [requarks/wiki#6998] [Help / Questions]
I've[ read the section on groups](https://docs.requarks.io/groups).  I understand what to do, but I don't know where it's located.  It's supposed to be in the page settings, right?  I also tried searching for videos, searching github, searching reddit.  I've worked in IT for 20 years so reasonably knowledgeable.

I'm running wiki.js as an app via cloudron.io on an AWS instance as a Docker container.  This isn't how I like doing things, but here we are.  (Despite having <500kB of content, no users, and setting free tier limitations AWS is trying to bill me ~$50/mo, but that's another story)  I understand how to set rules, but I can't find where the settings are supposed to be.  

I'm probably missing something obvious, hopefully someone can help me.

Wiki.js version: 2.5.301
Database engine: PostgreSQL 14.9
Node.js: 16.18.1
OS: AL 2 (running on AWS)
![wiki-js-page-rules](https://github.com/requarks/wiki/assets/150870811/3ea018c8-9150-451c-bd04-581dffbc3e30)

**Comment by fnafox** [comment]
Access permissions are defined per group in the Admin area: `/a/groups/`
First you define a group's global permissions in the permissions tab.
After that, granular rules can be defined in the page rules tab.
